swaygrab/json.c

```diff
@@ -12,7 +13,11 @@ static json_object *tree;
 void init_json_tree(int socketfd) {
 	uint32_t len = 0;
 	char *res = ipc_single_command(socketfd, IPC_GET_TREE, NULL, &len);
-	tree = json_tokener_parse(res);
+  enum json_tokener_error error = json_tokener_success;
```

Comment:
Tabs!

Reply:
I have no idea how that happened. I blame sublime text.